components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/JsGraphBuilder.java

```diff
@@ -245,6 +245,7 @@ private void handleStepOptions(StepConfig stepConfig, Map<String, String> stepOp
     protected void filterOptions(Map<String, Map<String, String>> authenticationOptions, StepConfig stepConfig) {
 
         Map<String, Set<String>> filteredOptions = new HashMap<>();
+        final String[] authenticatorNameResolver = new String[1];
```

Comment:
A string array is used here as non-final variables are not allowed to change inside lambda functions.